ibmdbpy4nps/ae/client code examples/stock_prediction_modelscoring_wlm_nps.py

```diff
@@ -4,15 +4,22 @@
 from ibmdbpy4nps import IdaDataBase, IdaDataFrame
 from ibmdbpy4nps.ae import NZFunGroupedApply
 
-dsn = 'bank'
+#dsn = 'bank'
+dsn='olaf'
 
-idadb = IdaDataBase(dsn, 'admin', 'password')
+#idadb = IdaDataBase(dsn, 'admin', 'password')
+idadb = IdaDataBase(dsn, 'pyuser','6VF6nvnP3QpBtkK')
```

Comment:
Real passwords shouldn't be included in a publicly accessible file. For the default password of the admin userid this is still acceptable, because it is already widely known. So I would replace the password string by something like 'xxxxx'.